spring-cloud-common-security-config-core/src/main/java/org/springframework/cloud/common/security/support/OAuth2AccessTokenProvidingClientHttpRequestInterceptor.java

```diff
@@ -56,11 +56,11 @@ public ClientHttpResponse intercept(HttpRequest request, byte[] body, ClientHttp
 			tokenToUse = this.staticOauthAccessToken;
 		}
 		else {
-			tokenToUse = TokenUtils.getAccessToken();
+			tokenToUse = null; //OAuth2TokenUtilsService.getAccessToken();
```

Comment:
Did you mean to use `OAuth2TokenUtilsService.getAccessToken();` instead of `null ?

Reply:
Fixed